.env.example

```
@@ -0,0 +1,6 @@
+ENVIRONMENT="name"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)